cosmos/loadtest/scripts/address.py

```
@@ -0,0 +1,24 @@
+import bech32
+
+
+class Address:
+    def __init__(self, addr):
+        if type(addr) == str:
+            hrp, data = bech32.bech32_decode(addr)
+            self._hrp = hrp
+            self._data = data
+        elif type(addr) == bytes:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)